cmd/main.go

```diff
@@ -149,3 +171,101 @@ func main() {
 
 	os.Exit(0)
 }
+
+func runMultishareReconciler(driverConfig *driver.GCFSDriverConfig) {
+	config, err := buildConfig(*kubeconfig)
+	if err != nil {
+		klog.Error(err.Error())
+		os.Exit(1)
+	}
+
+	kubeClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		klog.Error(err.Error())
+		os.Exit(1)
+	}
+	fsClient, err := clientset.NewForConfig(config)
+	if err != nil {
+		klog.Error(err.Error())
+		os.Exit(1)
+	}
+
+	factory := fsInformers.NewSharedInformerFactory(fsClient, resyncPeriod)
+	coreFactory := informers.NewSharedInformerFactory(kubeClient, resyncPeriod)
```

Comment:
let's keep the core factory in main function. It will be needed for other features like 100 shares